app/javascript/components/DragAndDropOnField.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { Component } from "react";
+import { Stage, Layer, Text, Group } from "react-konva";
+import PlayerChoice from "./PlayerChoice";
+
+export default class DragAndDropOnField extends Component {
+  setPlayers(playerNumber, team) {
```

Comment:
this function doesn't set anything, so name is misleading. 
```suggestion
  getListOfPlayers(playerNumber, team) {
```